test/runtests.jl

```diff
@@ -12,6 +12,9 @@ include("gc.jl")
 include("external.jl")
 include("swmr.jl")
 include("mmap.jl")
+if Sys.islinux()
```

Comment:
linux only?

Reply:
It failed on windows, so I temporaly disabled that in test.